standalone/instructlab-pipelines-with-dsp-on-rhoai-.md

```diff
@@ -0,0 +1,646 @@
+  secret_key:                 # The object store secret key (AWS Secret Access Key)
+  data_key:                   # The name of the tarball that contains SDG data.
+  endpoint:                   # The object store endpoint
+  region:                     # The region for the object store.
+  verify_tls:                 # Verify TLS for the object store.
+```
+
+Apply the yaml file to the cluster
+
+Create a ServiceAccount, ClusterRole and ClusterRoleBinding
+
+Provide access to the service account running the standalone.py script for accessing and manipulating related resources.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  namespace: <data-science-project-name/namespace>
+  name: secret-access-role
+rules:
+  - apiGroups: [""]
+    resources: ["pods", "pods/log", "configmaps", "persistentvolumeclaims", "secrets","events"]
+    verbs: ["get", "list", "watch", "create", "update", "delete"]
+
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs: ["get", "list", "create", "watch"]
+
+  - apiGroups: ["kubeflow.org"]
+    resources: ["pytorchjobs"]
+    verbs: ["get", "list", "create", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: secret-access-binding
+subjects:
+  - kind: ServiceAccount
+    name: <workbench-service-account-name> # created above in Step-2
+    namespace: <data-science-project-name/namespace>
+roleRef:
+  kind: ClusterRole
+  name: secret-access-role
+  apiGroup: rbac.authorization.k8s.io
+```
+Apply the yaml to the cluster.
+
+These are the required [RBAC configuration] which we are applying on the ServiceAccount.
+
+#### Configure Data Science Pipeline Server:
+
+From within the RHOAI dashboard, navigate to the "Data Science Pipelines" page and click "Configure pipeline server". This will present you with a form where you can upload the credentials for the S3 bucket you created in the previous step.
+
+<p align="center"><img src="assets/images/configure_pipeline_server.png" width=50%\></p>
+
+### Run the Pipeline
+
+Now that all the cluster requirements have been setup, we are ready to upload and run our InstructLab pipeline!
+
+#### Upload the Pipeline:
+
+Now we can go back to our RHOAI Data Science Pipelines dashboard and select **"Import pipeline"**. We recommend importing the pipeline yaml directly from the github repo using: `https://raw.githubusercontent.com/opendatahub-io/ilab-on-ocp/refs/heads/main/pipeline.yaml`
+<p align="center"><img src="assets/images/import_pipeline.png" width=50%\></p>
+
+#### Create a Run:
+Once the pipeline is uploaded we will be able to select **"Create run"** from the **"Actions"** dropdown. This will present us with a number of parameters we can set to customize our run. Click **"Create run"** at the bottom of the page to kick off your InstructLab pipeline.
+
+<p align="center"><img src="assets/images/parameters.png" width=50%\></p>
+
+#### Available Pipeline Parameters:
```

Comment:
What about adding two additional sections providing example pipeline parameters?

* Example pipeline parameters for a quick pipeline run:
Example parameters to do a quick test that can be run in aprox 90m using x NVIDIA XXX GPUs

* Example pipeline parametes for a  full pipeline run

@rimolive and @gmfrasca can probably provide these examples